scripts/build-linux.sh

```diff
@@ -61,6 +61,7 @@ if [ ${CPU} = mor1kx ]; then
 	# To rebuild, use https://ozlabs.org/~joel/litex_or1k_defconfig
 	ROOTFS_LOCATION="https://ozlabs.org/~joel/"
 	ROOTFS=${ARCH}-rootfs.cpio.gz
+	ROOTFS_MD5="3a254683a7b6f441a4acc0d4c555230a"
```

Comment:
Can you log a bug to move these into a config file? They should be part of https://github.com/timvideos/litex-buildenv/blob/master/scripts/settings.sh probably....